src/main/java/model/output/XmlWriter.java

```diff
@@ -0,0 +1,107 @@
+package model.output;
+
+import model.data.value.DataValue;
+import model.input.file.DataFile;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Class for writing an xml save file created by the user.
+ * DataFiles are added to the list and can then be written to the file
+ *
+ * @author Paul.
+ */
+public class XmlWriter {
+
+	private List<DataFile> dataFiles;
+	private Logger logger = Logger.getLogger("XmlWriter");
+	private Document document;
+
+	/**
+	 * Creates a new XmlWriter.
+	 */
+	public XmlWriter(List<DataFile> dataFiles) {
+		logger.log(Level.INFO, "Create new XMLWriter");
+		this.dataFiles = dataFiles;
```

Comment:
Dit is ook niet echt wat je wil. Nu kan bijvoorbeeld:
    List<DataFile> list = new ArrayList<>();
    new XmlWriter(list);
    list.add(bullshit);
doen. Je wilt dus new ArrayList<>(dataFiles) doen